### PR TITLE
Created new contact UI

### LINKED ISSUE #15: create new contact ui

### PR DESCRIPTION
**Pull request by nancy-binani**

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+
+import { Formik, Form as NeetoUIForm } from "formik";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx]: This is not a neetoUI module.
```suggestion
import { Formik, Form as FormikForm } from "formik";
```

**Review by yedhink — CHANGES_REQUESTED** [review]
@nancy-binani

**Comment by nancy-binani** [comment]
Fixed https://github.com/nancy-binani/wheel/pull/19#pullrequestreview-1122419229 please review.

**Comment by yedhink** [comment]
@nancy-binani You don't have to wait for my review. Go ahead and merge PRs on your own.

**Pull request merged**